packages/datadog_flutter_plugin/README.md

```diff
@@ -23,7 +23,7 @@ Your iOS Podfile must have `use_frameworks!` (which is true by default in Flutte
 
 ### Android
 
-On Android, your `minSdkVersion` must be >= 19, and if you are using Kotlin, it should be version >= 1.5.31.
+On Android, your `minSdkVersion` must be >= 19, and if you are using Kotlin, it should be version >= 1.6.21.
```

Comment:
side comment: Strictly speaking it should still be ok to have 1.4 or 1.5, because I think Kotlin should be able to keep binary compatibility for existing APIs 2 versions back, and during the dependency resolution Gradle will anyway update other Kotlin transitive dependencies to the most recent from the overall set of versions requested.

But let's keep it like that.